sql/sql_expression.cc

```diff
@@ -610,6 +610,10 @@ hasUnboundVariables() const
             return true;
     }
 
+    if (funcs.find("columnCount") != funcs.end()) {
```

Comment:
I understand how it fix your case, but thats not really the purpose of this method. If hasUnboundVariables is true, you would expect to have something in the variable or wildcards part of getUnbound.